integration_tests/features/support/steps.js

```diff
@@ -303,6 +303,87 @@ Given(
   }
 );
 
+Given(
+  /I have a SHA3 miner (.*) connected to seed node (.*)/,  
+  { timeout: 40 * 1000 },
+  async function (name, seed) {
+    //add the base_node
+    const node = this.createNode(name);
```

Comment:
This duplication is not great, but I didn't see it in the first review. Hoping to add some clippy like stuff to our JS